templates/todo/task_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+    <h1>Delete task</h1>
+
+    <p>Are you sure you want to delete the task {{ task }}</p>
```

Comment:
The `{{ task }}` objects as defined in models.py returns string `Content: <content>`. Here we should have just task content to make it easier to read.
```suggestion
    <p>Are you sure you want to delete the task {{ task.content }}</p>
```